prereq_cycle_test.go

```diff
@@ -1,68 +0,0 @@
-package ldclient
-
-import (
-	"log"
-	"os"
-	"testing"
-	"time"
-)
-
-func TestPrereqSelfCycle(t *testing.T) {
-	featureFlagA := newFlagWithPrereq("keyA", "keyA")
-	testFlag(t, "key", []FeatureFlag{featureFlagA})
-}
-
-func TestPrereqSimpleCycle(t *testing.T) {
-	featureFlagA := newFlagWithPrereq("keyA", "keyB")
-	featureFlagB := newFlagWithPrereq("keyB", "keyA")
-	testFlag(t, "keyA", []FeatureFlag{featureFlagA, featureFlagB})
-}
-
-func TestPrereqCycle(t *testing.T) {
-	featureFlagA := newFlagWithPrereq("keyA", "keyB")
-	featureFlagB := newFlagWithPrereq("keyB", "keyC")
-	featureFlagC := newFlagWithPrereq("keyC", "keyA")
-	testFlag(t, "keyA", []FeatureFlag{featureFlagA, featureFlagB, featureFlagC})
-}
-
-func testFlag(t *testing.T, key string, flags []FeatureFlag) {
```

Comment:
that's not an actual test, it is the common part DRYed up from the actual (removed) tests.